### PR TITLE
Converting Landscape Canvas tests to utilize the prefab system

### PR DESCRIPTION
**Pull request by jckand-amzn**

### CODE REVIEW

File: AutomatedTesting/Gem/PythonTests/largeworlds/landscape_canvas/EditorScripts/GraphUpdates_UpdateComponents.py [hunk 1]
```diff
@@ -37,79 +36,74 @@ def GraphUpdates_UpdateComponent():
...
     bush_spawner_id = hydra.find_entity_by_name('BushSpawner')
     flower_spawner_id = hydra.find_entity_by_name('FlowerSpawner')
     inverted_perlin_noise_id = hydra.find_entity_by_name('Invert')
 
     # Open Landscape Canvas and the existing graph
     general.open_pane('Landscape Canvas')
-    open_graph_id = landscapecanvas.LandscapeCanvasRequestBus(bus.Broadcast, 'OnGraphEntity', slice_root_id)
+    open_graph_id = landscapecanvas.LandscapeCanvasRequestBus(bus.Broadcast, 'OnGraphEntity', prefab_lc_root)
     Report.critical_result(Tests.existing_graph_opened, open_graph_id.IsValid())
+    #general.idle_wait_frames(60)
```
Comment (spham-amzn) [inline AutomatedTesting/Gem/PythonTests/largeworlds/landscape_canvas/EditorScripts/GraphUpdates_UpdateComponents.py:99]: Remove if not needed anymore

File: AutomatedTesting/Gem/PythonTests/largeworlds/landscape_canvas/EditorScripts/GraphClosed_OnLevelChange.py [hunk 2]
```diff
@@ -73,7 +72,7 @@ def GraphClosed_OnLevelChange():
     Report.result(Tests.graph_registered, graph_registered)
 
     # Open a different level, which should close any open Landscape Canvas graphs
-    general.open_level_no_prompt('WhiteBox/EmptyLevel')
```
Comment (cgalvan) [inline AutomatedTesting/Gem/PythonTests/largeworlds/landscape_canvas/EditorScripts/GraphClosed_OnLevelChange.py:76]: It looks like this was the only test left that was using this level, so can we delete `WhiteBox/EmptyLevel` now?

**Review by cgalvan — APPROVED** [review]

File: AutomatedTesting/project.json [hunk 3]
```diff
@@ -14,6 +14,7 @@
     "display_name": "AutomatedTesting",
     "icon_path": "preview.png",
     "external_subdirectories": [
+        "D:/Gems/EasyTerrain",
```
Comment (spham-amzn) [inline AutomatedTesting/project.json:17]: This is a hard coded absolute path, and a windows one at that. It needs to be relative

**Review by spham-amzn — CHANGES_REQUESTED** [review]
The absolute hardcoded path needs to be changed

File: AutomatedTesting/Gem/PythonTests/largeworlds/landscape_canvas/EditorScripts/GraphClosed_OnLevelChange.py (hunk 2 above)
Comment (jckand-amzn) [reply AutomatedTesting/Gem/PythonTests/largeworlds/landscape_canvas/EditorScripts/GraphClosed_OnLevelChange.py:76]: I'm not sure, this test simply opened a level that was available at the time, and wasn't made for this test. I'll leave the level handling up to the Whitebox team when they handle their prefab test conversions.

File: AutomatedTesting/Gem/PythonTests/largeworlds/landscape_canvas/EditorScripts/GraphUpdates_UpdateComponents.py (hunk 1 above)
Comment (jckand-amzn) [reply AutomatedTesting/Gem/PythonTests/largeworlds/landscape_canvas/EditorScripts/GraphUpdates_UpdateComponents.py]: Removed.

File: AutomatedTesting/Gem/PythonTests/largeworlds/landscape_canvas/EditorScripts/GraphClosed_OnLevelChange.py [hunk 4]
```diff
@@ -73,7 +72,7 @@ def GraphClosed_OnLevelChange():
...
+    general.open_level_no_prompt('Base')
```
Comment (mbalfour-amzn) [inline AutomatedTesting/Gem/PythonTests/largeworlds/landscape_canvas/EditorScripts/GraphClosed_OnLevelChange.py:75]: The comment says "open a different level", but isn't opening 'Base' opening the *same* level?  Does it matter?

**Review by mbalfour-amzn — APPROVED** [review]

File: AutomatedTesting/Gem/PythonTests/largeworlds/landscape_canvas/EditorScripts/GraphClosed_OnLevelChange.py (hunk 4 above)
Comment (jckand-amzn) [reply AutomatedTesting/Gem/PythonTests/largeworlds/landscape_canvas/EditorScripts/GraphClosed_OnLevelChange.py:75]: It's a different level. They have the same name but in different folders. hydra.open_base_level() opens Prefab/Base. This line just opens Base at the levels directory.

**Review by spham-amzn — APPROVED** [review]

**Pull request merged**